backend/tests/test_contact.py

```diff
@@ -56,3 +56,107 @@ def test_delete_a_contact(self):
         # deleting a contact that does not exist
         response = self.client.delete("/contact/{}".format(c_id))
         self.assertEqual(response.status_code, 404)
+
+    # testing creating a contact
+    def test_create_a_contact(self):
+        dummy_ct_id = uuid.uuid4()
+        dummy_ct = ContactType(
+            id=dummy_ct_id,
+            hex_colour="#ffffff",
+            type="dummy type",
+            description="dummy description",
+            contacts=[],
+        )
+        db.session.add(dummy_ct)
+        db.session.commit()
+
+        """
+        create_obj = {
+            "name": "dummy name",
+            "email": "dummy email",
+            "secondary_email": "dummy secondary email",
+            "cellphone": "dummy cell phone",
+            "role": "dummy role",
+            "organization": "dummy organization",
+            "neighbourhood": "dummy neighbourhood",
+            "contact_type": dummy_ct_id,
+        }
+        """
+
+        # creating a contact with valid inputs
+        response = self.client.post(
+            "/contact",
+            json={
+                "name": "dummy name",
+                "email": "dummy email",
+                "secondary_email": "dummy secondary email",
+                "cellphone": "dummy cell phone",
+                "role": "dummy role",
+                "organization": "dummy organization",
+                "neighbourhood": "dummy neighbourhood",
+                "contact_type": dummy_ct_id,
+            },
+        )
+        self.assertEqual(response.status_code, 200)
+
+        # check that the contact has been properly added to the correct contact type
+        contact_type = ContactType.query.filter_by(id=dummy_ct_id).first()
+        self.assertEqual(len(contact_type.contacts), 1)
```

Comment:
@dgiltz would prefer if you could verify the creation of your contact by using the `id` returned from your post request instead.